backend/api/views/simpleAuthorView.py

```diff
@@ -0,0 +1,19 @@
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+from rest_framework import status
+from django.shortcuts import redirect
+from ..serializers import AuthorSerializer
+
+@api_view(['POST'])
+def registerAuthor(request):
+    serializer = AuthorSerializer(data=request.data)
+    if serializer.is_valid():
+        new_author = serializer.save()
+        new_author.is_active = False
+        new_author.host = request.get_host()
+        new_author.url = "{}/author/{}".format(new_author.host, new_author.id.hex)
```

Comment:
signup_requests only have username, password and github_url.
It's a good idea to get host from the request, if you want, you can modify the admin action to do that.

Reply:
would the host we get from the request be the same as the host of the server, in the registration case @KevinHQi

Reply:
Not guaranteed to be the same, some comments in StackOverflow mentions it can be spoofed. I don't know if we should worry about that in our case. @AnkushSharma2698 
https://stackoverflow.com/questions/1451138/how-can-i-get-the-domain-name-of-my-site-within-a-django-template

Reply:
also, the function naming need to change. All author related sign ups should end with `request`, not just author.